src/EmailLogger.php

```diff
@@ -20,9 +20,12 @@ public function handle(MessageSending $event)
 
         $attachments = [];
         foreach ($message->getChildren() as $child) {
-            $attachmentPath = config('email_log.folder') . '/' . $messageId . '/' . $child->getFilename();
-            Storage::put($attachmentPath, $child->getBody());
-            $attachments[] = $attachmentPath;
+            //docs for this below: http://phpdox.de/demo/Symfony2/classes/Swift_Mime_SimpleMimeEntity/getChildren.xhtml
+            if(in_array(get_class($child),['Swift_EmbeddedFile','Swift_Attachment'])) {
```

Comment:
I don't know enough about this change, but am curious what this eliminates?

Reply:
I had some issues that the object returned was of type Swift_MimePart which doesn't have a `getFilename()` method defined, so this is a workaround.

Didn't have the time to investigate into this properly. I thinks it would be OK if it was checking for only Swift_Attachment.